resources/definitions/hellbot_magna_dual.def.json

```
@@ -0,0 +1,34 @@
+ {
+ 	"version": 2,
+ 	"name": "Hellbot Magna dual",
```

The name in the website is with capital letters.
```suggestion
 	"name": "Hellbot Magna DUAL",
```